config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
-
+  # scope :module => 'foreman_gutterball', :path => '/gutterball' do
+  #   resources :content_reports
   scope :module => 'foreman_gutterball', :path => '/gutterball' do
-    resources :content_reports, :only => [:index, :show] do
-      member do
-        get :generate
+    scope :module => 'api' do
+      scope :module =>  'v2' do
```

Comment:
So is it necessary to have the controller in the api/v2/ layer in the controllers folder?  If not, I can get rid of the scoping... i was trying to reduce the route name just to /gutterball rather than gutterball/api/v2/reports